api/v1alpha1/vip_webhook.go

```diff
@@ -89,10 +89,13 @@ func (r *Vip) validateVip() error {
 }
 
 func (r *Vip) validateAddresses() error {
-	_, _, err := net.ParseCIDR(r.Spec.Address)
+	_, n, err := net.ParseCIDR(r.Spec.Address)
```

Comment:
Instead, if a user sets "20.0.0.1/24" to create a VIP resource, could we replace it by "20.0.0./24"?
I think, this should be fix too for the subnet pools in the Trench and the subnets in the Attractor

Reply:
I would prefer suggesting 20.0.0./24 in the error message instead of replacing it without users' awareness.
I will add the same checker to attractor.spec.vlan-ipv4-prefix and attractor.spec.vlan-ipv6-prefix regarding the Attractor.
Do you mean the proxy EnvVar "NSM_SUBNET_POOLS" by "subnet pools in Trench"? But that is not exposed in the CR now.

Reply:
I thought it twice, maybe we should align to what most applications are doing. Maybe it's standard to replace it?